src/basic/process-util.c

```diff
@@ -1037,30 +1037,6 @@ bool is_main_thread(void) {
         return cached > 0;
 }
 
-_noreturn_ void freeze(void) {
```

Comment:
i am not convinced the reason to move this is sound. we use this from library code, in mount-util.c. Yes, that one is also in src/shared/,  so you get away with moving it, but still: it's fine to call this from some forked off child that some library maintains.

(but i don't really care)